Source/WebCore/fileapi/Blob.cpp

```diff
@@ -118,14 +119,18 @@ Blob::Blob(UninitializedContructor, ScriptExecutionContext* context, URL&& url,
     : ActiveDOMObject(context)
     , m_type(WTFMove(type))
     , m_internalURL(WTFMove(url))
+    , m_topOrigin(context ? context->topOrigin().data() : SecurityOriginData::createOpaque())
 {
+    ASSERT(context);
 }
 
 Blob::Blob(ScriptExecutionContext* context)
     : ActiveDOMObject(context)
     , m_size(0)
     , m_internalURL(BlobURL::createInternalURL())
+    , m_topOrigin(context ? context->topOrigin().data() : SecurityOriginData::createOpaque())
 {
+    ASSERT(context);
```

Comment:
I didn't find any call sites where the ScriptExecutionContext should be null, so this was only a sanity check.

Reply:
If call sites never passed null, we would take in a C++ reference, not a pointer. If something is a pointer in WebKit, always assume it can be null. If it really cannot, then update the parameter to be a c++ reference.